pkg/scheduler/api/helpers.go

```diff
@@ -60,6 +60,16 @@ func getTaskStatus(pod *v1.Pod) TaskStatus {
 	return Unknown
 }
 
+// PreemptibleStatus checks whether the task can be preempted
+func PreemptibleStatus(status TaskStatus) bool {
```

Comment:
fix the typo from `PreemptibleStatus` to `PreemptableStatus`

Reply:
Done